allensdk/api/cloud_cache/cloud_cache.py

```diff
@@ -339,6 +350,29 @@ def data_path(self, file_id) -> dict:
 
         return output
 
+    def get_file_attributes(self, file_id):
```

Comment:
pretty sure it returns `CacheFileAttributes` and not `dict` ? Can you add return type?

Reply:
Ah, was looking at my natural movie cache and saw what looked like a dict at first glace. Fixed.